src/sota_tools/deploy.h

```diff
@@ -13,9 +13,8 @@
  *                 are fetched from disk), or OSTreeHttpRepo (in which case
  *                 the objects will be pulled over https).
  */
-bool UploadToTreehub(const OSTreeRepo::ptr src_repo, const ServerCredentials& push_credentials,
-                     const OSTreeHash& ostree_commit, const std::string& cacerts, const bool dryrun,
-                     const int max_curl_requests);
+bool UploadToTreehub(const OSTreeRepo::ptr& src_repo, const ServerCredentials& push_credentials,
+                     const OSTreeHash& ostree_commit, const std::string& cacerts, bool dryrun, int max_curl_requests);
```

Comment:
What's wrong with `const`?

Reply:
Because parameter is passed by copy, not reffernce.

Reply:
There is still a semantic value in having a copied-by-value variable be const, as in, it's worth specifying that the value should not change.

Reply:
Hm, I think it is controversial issue. I did it because some check gives me a warning

Reply:
Indeed, I did not realize how controversial it was! This SO answer shares my perspective: https://stackoverflow.com/a/117557/289099. Since it doesn't programmaticallly matter, I'm not going to argue hard on this one.